src/test/java/org/folio/finc/config/ConfigFilesIT.java

```diff
@@ -121,4 +121,61 @@ public void checkThatWeCannotPostWithMissingIsil() {
         .then()
         .statusCode(400);
   }
+
+  @Test
+  public void checkThatDeletedFileCannotBeRetrieved() {
+    Response postResponse =
+      given()
+        .body(TEST_CONTENT.getBytes())
+        .header("X-Okapi-Tenant", TENANT_UBL)
+        .header("content-type", ContentType.BINARY)
+        .post(FINC_CONFIG_FILES_ENDPOINT + "?isil=" + isilUbl.getIsil())
+        .then()
+        .statusCode(200)
+        .extract()
+        .response();
+
+    String id = postResponse.getBody().print();
+
+    // DELETE
+    given()
+      .header("X-Okapi-Tenant", TENANT_UBL)
+      .delete(FINC_CONFIG_FILES_ENDPOINT + "/" + id)
+      .then()
+      .statusCode(204);
+
+    // Try to GET deleted file
+    given()
+      .header("X-Okapi-Tenant", TENANT_UBL)
+      .get(FINC_CONFIG_FILES_ENDPOINT + "/" + id)
+      .then()
+      .statusCode(404);
```

Comment:
Assertion could be merged into existing test case making `checkThatDeletedFileCannotBeRetrieved` obsolete.